sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -306,7 +306,7 @@ static int register_device()
           rc = az_iot_provisioning_client_register_get_publish_topic(
               &provisioning_client, register_publish_topic, sizeof(register_publish_topic), NULL)))
   {
-    printf("Failed to get MQTT register publish topic, az_result return code %04x\n", rc);
+    (void)printf("Failed to get MQTT register publish topic; az_result return code %04x .\n", rc);
```

Comment:
nit: being a bit of a grammar nazi here but I would prefer everywhere where semicolons are used that we use colons:
```suggestion
    (void)printf("Failed to get MQTT register publish topic: az_result return code %04x .\n", rc);
```